packages/strapi-helper-plugin/lib/src/utils/getFilterType.js

```diff
@@ -102,6 +102,21 @@ const getFilterType = type => {
           value: '_ncontains',
         },
       ];
+    case 'size':
```

Comment:
I am not sure this belong in this file as size is not an attribute type but more something you need to add in order to make your code works you should find another way to do it in the file where this is used.